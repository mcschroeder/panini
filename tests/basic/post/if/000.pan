f00 : {v:int|?}
f00 = if true then 1 else 2

f01 : {v:int|?}
f01 = if false then 1 else 2
