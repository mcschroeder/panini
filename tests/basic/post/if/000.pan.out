f00 : {v:ℤ | v = 1}
f01 : {v:ℤ | v = 2}